contrib/guix/libexec/build.sh

```diff
@@ -452,11 +452,13 @@ mv --no-target-directory "$OUTDIR" "$ACTUAL_OUTDIR" \
 
 (
     cd /outdir-base
-    {
-        echo "$GIT_ARCHIVE"
-        find "$ACTUAL_OUTDIR" -type f
-    } | xargs realpath --relative-base="$PWD" \
-      | xargs sha256sum \
+    # Get all of the generated output files and hash them without directory names.
+    # Because "find" prints filenames with a preceding "./", we need to pass the filenames into "basename" first.
```

Comment:
Generally I think it would be good to leave the preceding `./` path, because it is precise.
The verifying works with it on Linux, so the removing is not generally needed.
I guess it shall be removed to be compatible with OSs that have other directory naming conventions, e.g. like Microsoft products?
I would find a comment helpful, for what reason the `./` is needed to be removed.

Reply:
Typically SHA256SUMS do not contain paths to the files. Seeing the `./` may be confusing to users who are not familiar with paths.

Reply:
BTW [here](http://www.gnu.org/software/coreutils/manual/coreutils.html#sha2-utilities) (leading to the doc of `md5sum --check` invocation) is the doc of SHA256SUMS file:
_The input to this mode of md5sum is usually the output of a prior, checksum-generating run of ‘md5sum’._

Reply:
Yes, but many people do a visual comparison.